src/tests/unittests/asn1.c

```diff
@@ -234,6 +234,45 @@ static void torture_negative_int(void **state)
 	assert_int_equal(value, -224);
 }
 
+/*
+ * Test undefined behavior of too-large bit field (BIT STRING as an integer)
+ * https://oss-fuzz.com/testcase-detail/5764460018401280
+ *
+ * In this example, invalid "unused bytes" value was used
+ */
+static void torture_bit_field(void **state)
+{
+	/* Without the Tag and Length {0x03, 0x02} */
+	u8 data0[] = {0x07, 0x80};
+	/* Without the Tag and Length {0x03, 0x05} */
+	/* u8 data1[] = {0x20, 0x20, 0xff, 0xff, 0xff}; original data */
+	u8 data1[] = {0x20, 0xff, 0xff, 0xff, 0xff};
```

Comment:
test case with bits set but non-zero truncate missing, something like this:
`{0x01, 0xff}` for negative and `{0x01, 0xfe` for positive test

Reply:
Thank you. This was not handled in the code either. Added the testcase too.